capstone/src/main/java/com/step902020/capstone/Organization.java

```diff
@@ -0,0 +1,83 @@
+package com.step902020.capstone;
+
+import org.springframework.cloud.gcp.data.datastore.core.mapping.Entity;
+import org.springframework.cloud.gcp.data.datastore.core.mapping.Field;
+import org.springframework.data.annotation.Id;
+import java.util.*;
+
+@Entity(name = "organization")
+public class Organization {
+  @Id
+  Long datastoreId;
+
+  Long timestamp;
+
+  String name;
+
+  String email;
+
+  String university;
+
+  @Field(name="user-type")
+  String userType;
+  String description;
+  String image;
+  
+  public Organization() {
+
+  }
+  
+  public Organization(Long timestamp, String name, String email, String university, String userType, String description, String image) {
+    this.timestamp = timestamp;
+    this.name = name;
+    this.email = email;
+    this.university = university;
+    this.userType = userType;
+    this.description = description;
+    this.image = image;
+  }
```

Comment:
if a user wants to just update only one detail, is there a way to prefill the form so they dont have to rewrite all the details?